src/insns/gcperm_32bit.adoc

```diff
@@ -33,6 +33,9 @@ permission bits in `rd` are set to 0.
 [#gcperm_bit_field]
 include::../img/acperm_bit_field.edn[]
 
+NOTE: When {cheri_levels_ext_name} is not implemented, the `CL`, `SL`, `EL` and EL fields always report 0.
```

Comment:
EL and EL (with the latter copy also not in backticks)?

Reply:
thanks, fixed.